### DISCUSSION TITLE
configuration of  classes in the pretrained yolov8n

### DISCUSSION THREAD

**Discussion by MainManIsMID** [ultralytics/ultralytics#1138] [Q&A]
Is it possible to only detect certain Classes with the new Yolov8 pretrained model ?
I used the model in python and everything works though I would like to detect only persons

**Comment by glenn-jocher** [comment]
@MainManIsMID hello! Yes, you can certainly configure the YOLOv8 pretrained models to detect only specific classes, such as persons. To do this, you'll need to modify the class names list in your inference script to include only the classes you're interested in. For detecting only persons, you would set the class names list to include just the 'person' class. This way, when you run predictions, the model will only report detections for that class. For more detailed guidance on how to work with classes during inference, please refer to the 'Predict' section in our documentation. Happy detecting! 😊👤
